rocky/reports/report_types/ipv6_report/report.html

```diff
@@ -8,30 +8,34 @@
                 The IPv6 report provides an overview of the current IPv6 status of the
                 identified system. The table below shows whether the domain is reachable
                 over IPv6 or not. A green compliance check is shown if this is the case.
-                A red compliance cross is shown if no IPv6 address was detected.
+                A grey compliance cross is shown if no IPv6 address was detected.
             {% endblocktranslate %}
         </p>
         <div class="horizontal-scroll">
             <table>
-                <caption class="visually-hidden">{% translate "IPV6 overview:" %}</caption>
+                <caption class="visually-hidden">{% translate "IPV6 overview" %}</caption>
                 <thead>
                     <tr>
                         <th scope="col">{% translate "Domain" %}</th>
-                        <th scope="col">{% translate "System type" %}</th>
+                        {% if show_heading %}
```

Comment:
As far as I can see `show_heading` was meant to control whether the `h2` heading is shown or not... I think it is confusing if we reuse that variable to do other things, it would be better to rename the variable if we also want to use it for other things.